LoopKit/Service/StatefulPluggable.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Security.swift
```

Comment:
nit: Just a few remaining Security -> StatefulPluggable changes remaining in comments.